src/scripts/clipperUI/componentBase.ts

```diff
@@ -105,6 +105,26 @@ export abstract class ComponentBase<TState, TProps> {
 				} else if (e.which === Constants.KeyCodes.tab) {
 					// Since they are using the keyboard, revert to the default value of the outline so it is visible
 					element.style.outlineStyle = "";
+				} else if (e.which === Constants.KeyCodes.up) {
+					if (element.hasAttribute("data-" + ariaSet)) {
+							let nextPosInSet = parseInt(element.getAttribute("aria-posinset"), 10) - 1;
+							const buttons = document.querySelectorAll("a[data-" + ariaSet + "]");
+							if (nextPosInSet === 0) {
```

Comment:
Please see comments around this entire block of code. I don't think the suggestions were applied. E.g., by assigning nextPosInSet to buttons.length, wouldn't you loop back around to the last item in the set? We wish to do nothing in this case.